### PR TITLE
Milestone4 logic

### PR DESCRIPTION
**Pull request by Flakster**

### CODE REVIEW

**Comment by Flakster** [comment]
## This pull request includes

* Implementation of logic modules: calculate.js and operate.js
* Use of Big.js module
* Logic code for mathematical calculations

**Review by Forison — CHANGES_REQUESTED** [review]
## Status::Change Required❌

# 🙋‍♂️ @Flakster 

#### You have done a good job on this one so far, the logic of the application is organized as required here is one thing we have to address👨‍🎓.

* [ ] Check my comment below👇

#### That is it kindly make the change and submit a re-review💪

#### Happy coding👨‍💻

Addo Forison

File: src/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      result = number1 - number2;
+      break;
+    case 'x':
+      result = number1 * number2;
+      break;
+    case '÷':
+      result = number1 / number2;
+      break;
+    case '%':
+      result = number1 + 0.01;
```
Comment (Forison) [inline src/logic/operate.js]: * [ ] adding 0.01 to number1 won't convert the number to decimal but by multiplying, let us address this issue .

Comment (Flakster) [reply src/logic/operate.js]: You are absolutely right, it was a mistake

**Review by wathigo — APPROVED** [review]
## APPROVED :heavy_check_mark: 
Great job implementing the requested change.

**You may proceed to the next milestone.**

**Happy Coding :computer: 
Simon Wathigo**

**Pull request merged**